### PR TITLE
Added detectionfor Raspberry Pi 4 and Buster

### PR DESCRIPTION
**Pull request by darkdrgn2k**
- RPI4 detection
- Buster (and future) release change form testing to stable

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,11 +79,12 @@ List of tested hardware:
...
 | Genericx x86              | Debian 9                                                         |            |         |         |   |  Performance depended on underlying hardware. |
-| Raspberry Pi 3b+          | [Raspbian Lite](https://www.raspberrypi.org/downloads/raspbian/) | 405k, 119k | ~90 Mbps| 2       | 10/100/1000 | Eth only ~320mbps. Onboard wifi dual band |
-| Raspberry Pi 3b           | [Raspbian Lite](https://www.raspberrypi.org/downloads/raspbian/) | 350k, 100k | 89 Mbps | 2       | 10/100 | |
-| Raspberry Pi 2            | [Raspbian Lite](https://www.raspberrypi.org/downloads/raspbian/) | 145k,  55k | 39 Mbps | 2       | 10/100 | |
-| Raspberry Pi 1 A+         | [Raspbian Lite](https://www.raspberrypi.org/downloads/raspbian/) |  35k,   -  | ~9 Mbps | 1       | None   | |
-| Raspberry Pi 1 B+         | [Raspbian Lite](https://www.raspberrypi.org/downloads/raspbian/) |  51k,  22k | ~8 Mbps | 2       | 10/100 | |
+| Raspberry Pi 4b           | [Raspbian Lite](https://www.raspberrypi.org/downloads/raspbian/) | 650k, 230k | ~160 Mbps| 4       | 10/100/1000 | Onboard wifi dual band |
```
Comment (makeworld-the-better-one) [inline README.md]:
```suggestion
| Raspberry Pi 4B (2GB ram)           | [Raspbian Lite](https://www.raspberrypi.org/downloads/raspbian/) | 650k, 230k | ~160 Mbps| 4       | 10/100/1000 | Onboard wifi dual band |
```

Comment (makeworld-the-better-one) [reply README.md]: I am assuming you tested with the 2GB version, that's what I remember you saying.

File: scripts/cjdns/install [hunk 2]
```diff
@@ -40,6 +40,10 @@ case "$BOARD_MODEL" in
     i386)
         CJDNS_BUILD_CMD="sudo ./do"
     ;;
+    "raspberrypi4")
+        CJDNS_BUILD_CMD="sudo CFLAGS=\"-march=armv8-a+crc -mtune=cortex-a53 -mfpu=crypto-neon-fp-armv8 -mfloat-abi=hard -O2\" ./do"
```
Comment (makeworld-the-better-one) [inline scripts/cjdns/install]: The RPi4 has a Cortex-A**72** processor, I don't know if that would change the `-march` or `-mfpu` flags either.

File: README.md (hunk 1 above)
Comment (darkdrgn2k) [reply README.md]: Does it matter how much ram it has? i dont think it would change the workings of the pi.

**Pull request merged**